frontend/locales/en.json

```diff
@@ -630,5 +623,253 @@
+        "readMoreLink": "Read More"
+      },
+      "noSelection": "Select an entry on the left side to show the instructions."
+    },
+    "categories": {
+      "basic": "Basics",
+      "other": "Other",
+      "embedding": "Embedding",
+      "charts": "Charts & Diagrams"
+    },
+    "basics": {
+      "basicFormatting": {
+        "title": "Basic",
+        "description": "These are the basic markdown formatting rules.",
+        "example": "**Bold**\n__Bold__\n\n*Italic*\n_Italic_\n\n++Underline++\n~~Strikethrough~~\n\nSub~script~~\n\nSuper^script^\n\n==Marked=="
+      },
+      "abbreviation": {
+        "title": "Abbreviation",
+        "description": "Abbreviation definitions create tooltips for matching words. They can be placed defined in the document.",
+        "example": "*[HTML]: Hyper Text Markup Language\n*[W3C]:  World Wide Web Consortium\nThe HTML specification\nis maintained by the W3C."
+      },
+      "footnote": {
+        "title": "Footnotes",
+        "description": "Footnotes can be used to add extra information at the bottom of the page. They can be defined anywhere in the document.",
+        "example": "Here is a footnote reference,[^1] and another.[^longnote]\n\n[^1]: Here is the footnote.\n\n[^longnote]: Here's one with multiple blocks. Subsequent paragraphs are indented to show that they belong to the previous footnote."
+      },
+      "headlines": {
+        "title": "Headlines",
+        "hashtag": {
+          "title": "Hashtag",
+          "description": "Headlines can be used to structure your document into sections. Every headline creates a linkable anchor.",
+          "example": "# Headline 1\n\n## Headline 2\n\n### Headline 3\n\n#### Headline 4\n\n##### Headline 5\n\n###### Headline 6"
+        },
+        "equal": {
+          "title": "Equal",
+          "description": "An alternative form to define the first headline is to append a line of only equal-signs.",
+          "example": "Headline 1\n=========="
+        }
+      },
+      "code": {
+        "title": "Code",
+        "inline": {
+          "title": "Inline",
+          "description": "You can define an inline code block by wrapping it in backticks.",
+          "example": "This is `code`."
+        },
+        "block": {
+          "title": "Block",
+          "description": "You can define a code block by putting a line of three backticks before and after the code part.",
+          "example": "```\nthis is a code block\n```"
+        }
+      },
+      "lists": {
+        "title": "Lists",
+        "unordered": {
+          "title": "Unordered",
+          "description": "You can create unordered lists by prepending lines with dashes or asterisks.",
+          "example": "- A\n- B\n- C\n\n* A\n* B\n* C"
+        },
+        "ordered": {
+          "title": "Ordered",
+          "description": "You can create ordered lists by prepending lines with numbers followed by a dot. Ordered lists will always start with one.",
```

Comment:
Maybe add that the ordered list continues when no other paragraph is in between?

![image](https://user-images.githubusercontent.com/52606896/225651926-443fddf8-0c3e-4d63-b71e-8b022126103f.png)